### DISCUSSION TITLE
Problem Upload proof

### DISCUSSION THREAD

**Discussion by DanielRwx** [community/community#135532] [GitHub Education]
### Hi everyone,

Hello, i start studying Cybersecurity higher education, it is an online school, but i get error on trying upload my proof (I take photo of my school ID)

(Your application cannot be reviewed until you fix the following:

Please upload an image to help us understand why you are so far from campus (for example: proof that your classes are all remote or the address of a closer campus location we may not know about)
Please upload an image that is less than 1MB in size
Binary data can't be blank)

**Comment by odelyo** [comment]
It sounds like you're encountering multiple issues while trying to upload your proof of enrollment. Here’s how you can resolve each of the errors mentioned:

1. Proof of Remote Classes or Closer Campus:
Since you’re studying online, the system may require additional proof that your classes are remote.
Solution: Prepare a document or screenshot from your school’s online portal that clearly shows your enrollment in online classes. This could be a screenshot of your class schedule, an email from your school confirming that your classes are remote, or any other official document that shows your remote learning status.
2. Image Size Must Be Less Than 1MB:
The uploaded image of your school ID needs to be less than 1MB in size.
Solution:
Compress the Image: Use an online image compressor like [TinyPNG](https://tinypng.com/) or [JPEG-Optimizer](https://www.jpeg-optimizer.com/) to reduce the file size of your image.
Resize the Image: You can also resize the image using image editing software (like Paint on Windows or Preview on macOS) to make it smaller and reduce the file size.
3. Binary Data Can't Be Blank:
This error usually indicates that the file wasn’t properly uploaded or that the system didn’t recognize the file.
Solution:
Reupload the Image: After resizing and compressing your image, try uploading it again.
Check File Format: Ensure that the file is in a supported format (usually JPEG, PNG, or PDF).
Clear Browser Cache: If you’re still encountering this issue, try clearing your browser’s cache or using a different browser.
4. General Tips:
Check Your Internet Connection: Ensure you have a stable internet connection while uploading the file to prevent any interruptions.
Rename the File: Sometimes, renaming the file to something simple (like student_id.jpg) can help avoid issues.

**Comment by Bierchermuesli** [comment]


**Reply by Bierchermuesli** [reply]
i started over and somehow i got another form to upload a pdf

**Comment by parisppdo** [comment]
Hello my friend.
I am trying to do the same thing. However I have no idea what would be considered proof that my lessons are from distance. Any chance that you have an idea?

**Comment by abdelkarimain** [comment]
I hope it is related to this 
 https://github.com/orgs/community/discussions/37159
